notebooks/traj_based_model.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
Could you write a minimal version of this notebook that mostly imports stuff to train your model? Don't worry about adding any text or explanations, I can help with that later. I just want to have a clear picture of how your contributions in this PR work.

Reply:
Do you still want this? I can also do this. I think now that I removed a lot of changes it should be simpler to follow.